app/assets/stylesheets/active_admin/print.scss

```diff
@@ -228,61 +153,44 @@ table {
   }
 }
 
-// Sidebars
-#filters_sidebar_section {
-  display: none;
-}
-
 // Forms
-form {
-  fieldset {
-    border-top: thick solid #ccc;
-    padding-top: 2em;
-    margin-bottom: 2em;
+fieldset {
+  border: none;
+  margin: 0 0 2em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 2em` instead of `0 0 2em 0`